test/controllers/user.test.js

```diff
@@ -66,3 +67,52 @@ describe('usersController.signUp', () => {
       });
   });
 });
+
+describe('usersController.signIn', () => {
+  it('Log in with previously created user', () => {
+    userAttributes.password = 'password1923';
+    return request
+      .post('/users')
```

Comment:
Also, user creation is a task shared between all 3 test and should be in a `beforeEach`: https://jestjs.io/docs/en/setup-teardown